src/status_im/ui/screens/chat/state.cljs

```diff
@@ -0,0 +1,3 @@
+(ns status-im.ui.screens.chat.state)
+
+(defonce viewable-item (atom nil))
```

Comment:
shouldn't this be cleaned after logout?

Reply:
I can probably clean it up as we land on a chat, it's probably safer, but should get reset as soon as we load messages